docs/docs/fallback-handoff.mdx

```diff
@@ -231,80 +215,147 @@ class ActionDefaultFallback(Action):
-```
+In some cases, you will want to attempt to disambiguate the user's message by asking
+clarifying questions. The Two-Stage-Fallback is made to handle low NLU confidence in multiple stages
+using the following sequence:
+
+- If an NLU prediction has a low confidence score, the user is asked to affirm
+  the classification of the intent.  (Default action:
+  `action_default_ask_affirmation`)
+
+    - If they affirm by sending a message with high NLU confidence (e.g. by pressing
+      a button), the story continues as if the intent was classified
+      with high confidence from the beginning.
+    - If they deny by sending a message with the intent `out_of_scope`, the user is
+      asked to rephrase their message.
 
-We'll need to add NLU data for the `out_of_scope` intent as well:
+- Rephrasing  (default action: `action_default_ask_rephrase`)
 
-```md
-## intent:out_of_scope
-- I want to order food
-- What is 2 + 2?
-- Who's the US President?
-- I need a job
+    - If the classification of the rephrased intent was confident, the story
+      continues as if the user had this intent from the beginning.
+    - If the rephrased intent was not classified with high confidence, the user
+      is asked to affirm the classified intent.
+
+- Second affirmation  (default action: `action_default_ask_affirmation`)
+
+    - If they affirm by sending a message with high NLU confidence (e.g. by pressing
+      a button), the story continues as if the user had this intent from the beginning.
+    - If the user denies by sending a message with the intent `out_of_scope`, the
+      original intent is classified as the specifies `deny_suggestion_intent_name`,
+      and an ultimate fallback action `fallback_nlu_action_name` is
+      triggered (e.g. a handoff to a human).
+
+
+The Two-Stage-Fallback can be enabled using the following steps:
+
+#### 1. Updating the configuration
+
+Add `FallbackClassifier` to your pipeline and the [RulePolicy](./policies.mdx#rule-policy)
+to your policy configuration:
+
+```yaml
+pipeline:
+# other components
+- name: FallbackClassifier
+  confidence_threshold: 0.7
+
+policies:
+# other policies
+- RulePolicy
 ```
 
-And finally we'll add a response to our `domain.yml` file:
+
+#### 2. Defining the fallback responses
+
+Rasa Open Source provides default implementations for
+`action_default_ask_affirmation` and `action_default_ask_rephrase`.
+The default implementation of `action_default_ask_rephrase` utters
+the response `utter_ask_rephrase`, so make sure to specify this
+response in your domain file:
 
 ```yaml
 responses:
-  utter_out_of_scope:
-  - text: Sorry, I can't handle that request.
+  utter_ask_rephrase:
+  - text: I'm sorry, I didn't quite understand that. Could you rephrase?
 ```
 
-We can now re-train, and test this addition
+:::note
+The implementations of `action_default_ask_affirmation` and `action_default_ask_rephrase`
+can be overwritten using [Custom Actions](./actions.mdx#custom-actions).
```

Comment:
can we actually link to the default actions customisation page here? And say "to customise the behaviour of this policy..."